modules/persistent/arrow-persistent-data-structures/src/main/kotlin/arrow/persistent/extensions/persistentmap.kt

```diff
@@ -0,0 +1,53 @@
+package arrow.persistent.extensions
```

Comment:
All these extensions need to be in another module `arrow-persistent`, you need to compile `arrow-persistent-data` and check what you need in your tests. 

If you write TestLaws, they need to be in `arrow-persistent-data` - Just testCompile what you need.

Reply:
Can you elaborate a bit further? If I understand correctly we need to rename `arrow-persistent-data-structures` to `arrow-persistent-data`, but I'm not sure what you are trying to say here. Do we need 2 different modules?

Reply:
We've discussed this with @nomisRev on Slack and we agreed that it is fine as it stands now (since I inlined the `@higherkind` boilerplate and the whole thing will be rendered unnecessary after KEEP-87.